spec/features/data_migrate_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "spec_helper"
+
+RSpec.describe 'DataMigrate' do
```

Comment:
The first argument to describe should be the class or module being tested.

Reply:
Changing this would put this file in a separate folder from the rest of the features, I would keep it as it is